capstone/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -17,4 +17,9 @@ private Constants() {}
   static final String OFFICER_PROP = "officers";
   static final String ANNOUNCE_PROP = "announcements";
   static final int LOAD_LIMIT = 10;
+  static final String CLUB_PROP = "club";
+  static final String ANNOUNCEMENT_PROP = "Announcement";
```

Comment:
Is there a need for two announcement-prop constants with the only very slight differences in spelling? Where would you use one vs the other?

Reply:
The ANNOUNCEMENT_PROP constant is used to store / access the announcement entities in Datastore, but I'm not sure the other one is used? @K-Shao can you confirm that the ANNOUNCE_PROP is not used anywhere? If it's not needed, I can go ahead and remove it

Reply:
Ah, I see where it's being used - ANNOUNCE_PROP is used to store the announcements in the Club entities in Datastore. I believe Megan will be removing the announcements from the club entity since the announcement entities include which club the announcement is for, so that constant should be removed soon

Reply:
Yup! I think it's from when we were structuring it the other way, with announcements in clubs